source/Camera/Basic/CaptureAssistant.m

```diff
@@ -0,0 +1,82 @@
+try
+    app = zividApplication; 
+
+    disp('Connecting to camera')
+    camera = connectCamera(app);
+    
+    suggestSettingsParameters = Zivid.NET.('CaptureAssistant+SuggestSettingsParameters')(Zivid.NET.Duration.FromMilliseconds(1200),Zivid.NET.('CaptureAssistant+AmbientLightFrequency.none'));
```

Comment:
This should be explained somewhere, but probably not in the code??

Reply:
I guess it could be added somthing like:
% this is due to limitations in matlab support of .net nested classes. Link to limitations www.??

Reply:
Yes, in the capture tutorial. One such doesn't exist yet for Matlab (that is the next step after this PR). Here is how it looks like for python: https://github.com/zivid/python-samples/blob/master/source/camera/basic/CaptureTutorial.md